src/SFA.DAS.EmployerCommitmentsV2.Web/Models/Cohort/LegalEntitySignedAgreementViewModel.cs

```diff
@@ -1,19 +1,13 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Threading.Tasks;
-
-namespace SFA.DAS.EmployerCommitmentsV2.Web.Models.Cohort
+﻿namespace SFA.DAS.EmployerCommitmentsV2.Web.Models.Cohort
 {
     public class LegalEntitySignedAgreementViewModel
     {
-        public string HashedAccountId { get; set; }
```

Comment:
I actually prefer `HashedAccountId` is this change just for consistency?

Reply:
yes changed  for consistency :-)